tutorials/abap-environment-s4hanacloud-service-consumption/abap-environment-s4hanacloud-service-consumption.md

```diff
@@ -46,155 +38,135 @@ author_profile: https://github.com/mervey45
 
       ![Create Custom CDS View in SAP S/4HANA Cloud](cds2.png)
 
-  3. Name your CDS view `YY1_SALESORDERITEMCUBEXXX` and click **Add** and select **Add Primary Data Source**.
+  3. Name your CDS view `YY1_SALESORDERITEMCUBEXXX` and label. Select **External API** as scenario and click **Create**.
 
       ![Create Custom CDS View in SAP S/4HANA Cloud](cds3.png)
 
-  4. Search for **`I_SalesOrderItemCube`**, select it and click **Add**.
+  4. Search for **`I_SalesOrderItemCube`** and select it.
 
       ![Create Custom CDS View in SAP S/4HANA Cloud](cds4.png)
 
-  5. Click **Add** and select **Add Associated Data Source**.
+  5. Click **Add** and select **Associated Data Source**.
 
       ![Create Custom CDS View in SAP S/4HANA Cloud](cds5.png)
 
-  6. Search for **`I_BusinessUser`**, select it and click **Add**.
+  6. Search for **`I_BusinessUser`**, select it and click **OK**.
 
       ![Create Custom CDS View in SAP S/4HANA Cloud](cds6.png)
 
-  7. Select **Yes** and move on.
+  7. Select the butterfly symbol to add a join condition.
 
       ![Create Custom CDS View in SAP S/4HANA Cloud](cds7.png)
 
-  8. Select **External API** and click on the butterfly symbol.
+  8. Click **Add**.
 
       ![Create Custom CDS View in SAP S/4HANA Cloud](cds8.png)
 
-  9. In the field section search for following fields and add them to your CDS view.
-     - `SalesOrder`
-     - `SalesOrderItem`
-     - `_CreatedBy.BusinessPartner`
-     - `_SalesOrder.OverallSDProcessStatus`
-     - `CreationDate`
-     - `NetAmountInDisplayCurrency`
-     - `DisplayCurrency`
+  9. Search for `userid` and select it.
 
-      ![Create Custom CDS View in SAP S/4HANA Cloud](cds9.png)
+      ![Create Custom CDS View in SAP S/4HANA Cloud](cdsx.png)
 
-  10. Go to properties and check your result.
+ 10. Add a value to your join condition, therefore select the **field symbol** under Value.
 
       ![Create Custom CDS View in SAP S/4HANA Cloud](cds10.png)
 
-  11. Switch to Parameters and set following manual default values:
-      - `P_ExchangeRateType`: EUR
-      - `P_DisplayCurrency`: M
+ 11. Search for `CreatedByUser` under `I_SalesOrderItemCube` and select it.
+
+       ![Create Custom CDS View in SAP S/4HANA Cloud](cds11.png)
+
+ 12. Now your join condition is complete. Click **Close**.
 
-      Click **Publish**.
+       ![Create Custom CDS View in SAP S/4HANA Cloud](cds12.png)
 
-      ![Create Custom CDS View in SAP S/4HANA Cloud](cds11.png)
+ 13. Navigate to **Parameters** and add manual parameters for the exchange rate type and display currency.
+
+       ![Create Custom CDS View in SAP S/4HANA Cloud](cds16.png)
+
+ 14. Navigate to **Elements**, select **Add** > **Elements**.
+
+       ![Create Custom CDS View in SAP S/4HANA Cloud](cds13.png)
+
+ 15. Search for `BusinessPartner`, select it and click **OK**.
+
+       ![Create Custom CDS View in SAP S/4HANA Cloud](cds14.png)
+
+ 16. Add also following elements to your CDS view:
```

Comment:
Add also the following elements to your CDS view.

or 
Also, add the following elements to your CDS view.

Anyway, please add "the".

Reply:
I updated this.